flink-runtime/src/main/java/org/apache/flink/streaming/runtime/operators/sink/SinkWriterOperator.java

```diff
@@ -178,7 +214,7 @@ public void processElement(StreamRecord<InputT> element) throws Exception {
     @Override
     public void prepareSnapshotPreBarrier(long checkpointId) throws Exception {
         super.prepareSnapshotPreBarrier(checkpointId);
-        if (!endOfInput) {
+        if (!this.endOfInput) {
```

Comment:
Nit: Why `this` here we do not use inside the other methods?

Reply:
Yes, I'll remove.